src/test/java/test/ContainersTest.java

```diff
@@ -27,7 +27,7 @@ void initHelm(TestInfo testInfo) {
     }
 
     @ParameterizedTest
-    @EnumSource
+    @EnumSource(value = Product.class, names = {"agent"}, mode = EnumSource.Mode.EXCLUDE)
```

Comment:
There are a fair bit of Exclusions in the unit tests. Would it make sense to leave out bamboo agent out of the Product enum? It is not really a product anyway and I think going forward they will probably diverge more than converge.
You have a specialised test for it and it can be potentially expanded if we are missing something critical from the other parametrised tests.